.env.example

```
@@ -11,6 +11,9 @@ DATABASE_URL=sqlite:///db.sqlite3
 
 # Email Configuration for OTP and Password Reset Emails
 # Use Gmail App Passwords (not normal Gmail password)
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+# 
+# Note: The email backend automatically defaults to the console backend when DEBUG=True 
+# (printing emails to your terminal). You can explicitly set EMAIL_BACKEND here to override this.
+EMAIL_BACKEND= 'django.core.mail.backends.smtp.EmailBackend'
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)